aws_lambda_powertools/utilities/parser/compat.py

```diff
@@ -0,0 +1,28 @@
+def disable_pydantic_v2_warning():
+    """
+    Disables the Pydantic version 2 warning by filtering out the related warnings.
+
+    This function checks the version of Pydantic currently installed and if it is version 2,
+    it filters out the PydanticDeprecationWarning and PydanticDeprecatedSince20 warnings
+    to suppress them.
+
+    Note: This function assumes that Pydantic is already imported.
```

Comment:
Is this still valid? I can see pydantic being loaded later on anyways "from pydantic"